### ISSUE TITLE
MakeCredentialOptions.parameters needs more precise name

### ISSUE THREAD

**Issue by equalsJeffH** [w3c/webauthn#473]
[MakeCredentialOptions.parameters](https://w3c.github.io/webauthn/#dictdef-makecredentialoptions) used to be named (a while back) "cryptoParameters".

Unfortunately, the latter name is also not-so-great because [MakeCredentialOptions.parameters](https://w3c.github.io/webauthn/#dictdef-makecredentialoptions) is of type `sequence<PublicKeyCredentialParameters>` where:
```
dictionary PublicKeyCredentialParameters {
    required PublicKeyCredentialType  type;
    required AlgorithmIdentifier   algorithm;
};
```
So, given that we've adopted a spell-it-all-out-fully naming regime, I suppose a new suggested name meeting such requirements is:  `MakeCredentialOptions.publicKeyCredentialParameters`.

**Comment by jyasskin** [comment]
We only absolutely need spell-it-all-out names at the top level. For members, shorter names are nice, using the disambiguation of the top-level type name.

**Comment by equalsJeffH** [comment]
thx @jyasskin -- so this would be acceptable? 
```
dictionary MakeCredentialOptions {
    [...]
    required sequence<PublicKeyCredentialParameters>   pubKeyCredParams;

    [...]
};

```

**Comment by selfissued** [comment]
+1 on fixing this

**Comment by herrjemand** [comment]
+1

**Comment by nadalin** [comment]
Create PR and merge

**Comment by selfissued** [comment]
I will create a PR using the name pubKeyCredParams

**Comment by selfissued** [comment]
This was addressed by PR #511
